docker-compose.yml

```diff
@@ -2,26 +2,49 @@ version: '3'
 
 services:
 
-  clamav_api:
-    image: quay.io/ukhomeofficedigital/docker-clamav-rest:1.0.0
+  database:
+    image: postgres:10.4
+    container_name: database
+    env_file:
+      - .env.dev
     ports:
-      - "8080:8080"
-    environment:
-      - CLAMD_HOST=clamav
+      - "5432:5432"
+
+  api:
+    image: api
+    container_name: api
+    env_file:
+      - .env.dev
     depends_on:
-      - clamav
+      - database
+    ports:
+      - "5000:5000"
 
   clamav:
     image: quay.io/ukhomeofficedigital/clamav:v1.6.1
+    container_name: clamav
     ports:
-     - "3310:3310"
+      - "3310:3310"
 
-  db:
-    image: postgres:10.4
-    restart: always
-    ports:
-     - "5431:5432"
+  clamav-api:
+    image: quay.io/ukhomeofficedigital/docker-clamav-rest:1.0.0
+    container_name: clamav-api
     environment:
-     - POSTGRES_PASSWORD=password
-     - POSTGRES_USER=john
-     - POSTGRES_DB=db
+      - CLAMD_HOST=clamav
+    depends_on:
+      - clamav
+    ports:
+      - "8080:8080"
+
+  node:
+    build: .
+    container_name: node
```

Comment:
The docker-compose works fine and I can see all 5 docker containers up locally. However I don't get the registration email in my inbox, so might be missing some step here.

Reply:
I cannot replicate locally - need to cooperate with you on that.

Reply:
Issue solved. Docker image running had time skew. 
Docker service restart fixed the time skew issue, registration and authentication emails are now being received.